test/specs/farmer.e2e.js

```diff
@@ -5,44 +5,52 @@ import knockoutQuestionsPage from 'page-objects/knockout.questions.page.js'
 import LandParcelsPage from 'page-objects/land.parcels.page.js'
 import ActionsPage from 'page-objects/actions.page.js'
 import FundingDetailsPage from 'page-objects/funding.details.page.js'
+import AgreementNamePage from 'page-objects/agreement.name.page.js'
 import * as assert from 'node:assert'
 
 describe('Given farmer is eligible for funding', () => {
   describe('When farmer goes through the land grands application', () => {
     const parcel = 'SX0679-9238'
+    const agreementName = 'Test Agreement'
     const action = 'CMOR1'
     const area = 1
     const totalApplicationValue = '£100.98'
     it('Then the farmer is shown the landing page', async () => {
       await HomePage.open()
       await expect(browser).toHaveTitle(
-        'Start page - Find Funding for Land or Farms - GOV.UK'
+        'Start page | Manage land-based actions'
       )
     })
 
     it('Then the farmer is shown the eligibility questions', async () => {
       await HomePage.clickButton('Start now')
 
       await expect(browser).toHaveTitle(
-        `Do your digital maps show the correct land details? - Find Funding for Land or Farms - GOV.UK`
+        `Do your digital maps show the correct land details? | Manage land-based actions`
       )
     })
 
-    it('Then the farmer is shown the land parcels for this application', async () => {
+    it('Then the farmer is asked to enter agreement name', async () => {
       await knockoutQuestionsPage.selectRadioButtonByValue('true')
       await knockoutQuestionsPage.clickButton('Continue')
+      await expect(browser).toHaveTitle(
+        `Enter your agreement name | Manage land-based actions`
+      )
+    })
 
+    it('Then the farmer is shown the land parcels for this application', async () => {
+      await AgreementNamePage.enterAgreementName(agreementName)
+      await AgreementNamePage.clickButton('Continue')
       await expect(browser).toHaveTitle(
-        `Select Land Parcel - Find Funding for Land or Farms - GOV.UK`
+        `Select Land Parcel | Manage land-based actions`
       )
     })
 
     it('Then the farmer has option to select the action for the land parcel', async () => {
       await LandParcelsPage.selectRadioButtonByValue(parcel)
       await LandParcelsPage.clickButton('Continue')
-
       await expect(browser).toHaveTitle(
-        `Select Land Actions - Find Funding for Land or Farms - GOV.UK`
+        `Select Land Actions | Manage land-based actions`
```

Comment:
is its worth maybe creating a variable for the service name so its a single place that needs to be updated?